content/api/vision/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Vision
+---
+
+Your webcam is how your robot will see markers.
+
+## Searching for markers
+Assuming you have a webcam connected, you can use `r.camera.see()` to take a picture. The software will process the picture and returns a list of the markers it sees, sorted by distance, where the first element of the array is the closest marker to the camera.
+
+```python
+markers = r.camera.see()
+```
+
+## Marker properties
+The markers in the list have some useful properties:
+
+- `is_wall_marker()` - returns whether or not the marker is a [wall marker](marker-ids/#wall-markers).
+- `is_token_marker()` - returns whether or not the marker is a [token marker](marker-ids/#token-markers).
+- `id` - returns the [id](marker-ids) of the marker.
+- `pixel_centre` - returns the pixel coordinates of the centre of the marker.
+- `polar` - returns details of the position of the marker in the [Polar](coordinates/#polar-coordinates) coordinate system.
+- `cartesian` - returns details of the position of the marker in the [Cartesian](coordinates/#cartesian-coordinates) coordinate system.
+
+{{% notice tip %}}
+The markers' position can be represented using both the [Polar](coordinates/#polar-coordinates) and [Cartesian](coordinates/#cartesian-coordinates) coordinate system.
```

Comment:
The grammar doesn't quite agree at the start of this sentence. Please change to either "marker's position" (the position of a single marker) or "markers' positions" (the positions of many markers).

Also, "system" -> "systems", since we're talking about more than one coordinate system.